bootstrap/bootstrap.sh

```diff
@@ -28,8 +29,32 @@ sudo apt-get update
 sudo apt-get -y dist-upgrade
```

Comment:
does the pi need to be rebooted at this point?  or is it safe to do the rest of the script first, and then reboot the pi?